index.html

```diff
@@ -1751,12 +1848,10 @@ <h5 id="Psionic Traits">Psionic Traits</h5>
                <td>Optic Separation
                </td>
                <td>
-                  <wip>
-                     Detach your vision from your body, and your mental eye can move up to
-                     50ft including movement through all physical objects and walls. 
-                     You may continue to move this eye for 1 AP on subsequent turns without 
-                     spending additional ability uses.
-                  </wip>
+                  You may detach your vision from your body, and your mental eye can move up to
```

Comment:
Much more balanced than phase shift imo and potentially leads to role play elements.  Maybe adding something which allows the mental eye to be detected could increase the risk vs reward of this?